.github/workflows/conformance-ingress.yaml

```diff
@@ -417,6 +417,17 @@ jobs:
             -enable-http-debug \
             -stop-on-failure
 
+      - name: Run basic CLI tests
+        shell: bash
+        run: |
+          mkdir -p cilium-junits
+          cilium connectivity test --include-unsafe-tests --collect-sysdump-on-failure \
+            --sysdump-hubble-flows-count=1000000 --sysdump-hubble-flows-timeout=5m \
+            --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>" \
+            --junit-file "cilium-junits/${{ env.job_name }} (${{ join(matrix.*, ', ') }}).xml" \
```

Comment:
The `job_name` environment variable is not defined, we can either change it with a hard-coded value or add an environment variable to the job definition.
Also, ${{ join(matrix.*, ", "}} is not necessary here, it will add too many items to the name, we can use ${{ matrix.name }} since it is unique.
`--junit-file "cilium-junits/<env.job_name or "connectivity-test">-${{ matrix.name }}"`